recirq/qaoa/placement.py

```diff
@@ -49,14 +49,14 @@ def calibration_data_to_graph(calib_dict: cg.Calibration) -> nx.Graph:
     return err_graph
 
 
-def _qubit_index_edges(device):
+def _qubit_index_edges(device: cirq.Device):
     """Helper function in `_device_to_tket_device`"""
-    dev_graph = ccr.gridqubits_to_graph_device(device.qubit_set())
+    dev_graph = ccr.gridqubits_to_graph_device(device.metadata.qubit_set())
```

Comment:
doesn't this give typing warnings about `metadata` might be `None`?

Reply:
Done.